Src/Plugins/Security/PFXCryptographicKeyFileValidator.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+
+namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security
+{
+    internal static class PFXCryptographicKeyFileValidator
+    {
+#pragma warning disable IDE0060 // Remove unused parameter
+        public static string IsValid(
+            ref string matchedPattern,
+            ref Dictionary<string, string> groups,
+            ref bool performDynamicValidation,
+            ref string failureLevel,
+            ref string fingerprint)
+        {
+#pragma warning restore IDE0060
+            fingerprint = $"[cert={matchedPattern}]";
```

Comment:
>fingerprint [](start = 12, length = 11)

What's the input matched pattern here? Is it just the file name? We could use that, I suppose, but better would be to find an identifier, like the public key token, that is consistent no matter what the binary container file is named.

Reply:
its the fullpath. c:\path\file.pfx, for example

---
In reply to: [554470486](https://github.com/microsoft/sarif-pattern-matcher/pull/96#discussion_r554470486) [](ancestors = 554470486)

Reply:
changed to [thumbprint=value]

---
In reply to: [554470631](https://github.com/microsoft/sarif-pattern-matcher/pull/96#discussion_r554470631) [](ancestors = 554470631,554470486)